Expose public path in workflow env

ovineq opened the pull request:
In order for next to build on github actions

aronhoyer commented on .github/workflows/checks.yaml:
```diff
@@ -56,3 +56,5 @@ jobs:
         run: |
           npm run seed:mock:all
           npm run build
+        env:
```
doesn't this need to be set **before** `run`? 🤔

ovineq replied:
Didn't think it mattered as long as it's in the same scope/indentation. But testing now. 👍🏻

ovineq commented:
> I feel like we should find a different solution to this and `.env` should be added to `.gitignore`. Allowing the file to be pushed to the remote opens up for someone accidentally committing their env config.
> 
> @nilsml had some experience with reading next js env vars from a different project, so maybe link up with him smile

Removed the `.env` and added it to the ignore list. 
Using localhost as the environment variable works, and should suffice for this workflow.
We can revisit the usage of secrets when we're setting up environments.

aronhoyer reviewed (approved):
looks great!
